packages/theme-chalk/src/upload.scss

```diff
@@ -334,36 +363,34 @@
       }
     }
 
-    .#{$namespace}-upload-list__item-actions {
+    .#{bem('upload-list', 'item-actions')} {
       position: absolute;
       width: 100%;
       height: 100%;
       left: 0;
       top: 0;
       cursor: default;
-      text-align: center;
+
+      display: inline-flex;
+      justify-content: center;
+      align-items: center;
```

Comment:
**Question:** can we extract this as a function in this file so that we don't have to write the same code over again just include them would be ok